xml/System.Globalization/CompareInfo.xml

```diff
@@ -256,12 +256,14 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="string1">To be added.</param>
-        <param name="string2">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="string1">The first string to compare.</param>
+        <param name="string2">The second string to compare.</param>
+        <param name="options">The <see cref="T:System.Globalization.CompareOptions" /> to use during the comparison.</param>
```

Comment:
does the documentation care to mention this parameter is defaulted to some value?

Reply:
You're right, we can mention that it has a default value:

```suggestion
        <param name="options">An optional <see cref="T:System.Globalization.CompareOptions" /> enumeration value to use during the comparison. The default value is <see cref="F:System.Globalization.CompareOptions.None" />.</param>
```